src/FactSystem/FactGroup.cc

```diff
@@ -116,7 +116,8 @@ FactGroup* FactGroup::getFactGroup(const QString& name)
         factGroup = _nameToFactGroupMap[camelCaseName];
         QQmlEngine::setObjectOwnership(factGroup, QQmlEngine::CppOwnership);
     } else {
-        qWarning() << "Unknown FactGroup" << camelCaseName;
+        qWarning() << "Unknown FactGroup CC" << camelCaseName;
```

Comment:
Could you change this to be "Unknown FactGroup (camelCaseName)"? That will make it way more clear.

Reply:
Sorry what I meant was to change to:
```qWarning() << "Unknown FactGroup (camelCaseName)" << camelCaseName;```